controllers/errors.controllers.js

```diff
@@ -0,0 +1,15 @@
+exports.handle400s = (err, request, response, next) => {
+    if (err.code === '23502'){
+        response.status(400).send({msg: "Bad request"})
+    } else {
+        next (err)
+    }
+}
+
+exports.handleCustomErrors = (err, request, response, next) => {
+    if (err.status && err.msg){
+        response.status(err.status).send({msg:err.msg})
+    } else {
+        next(err)
+    }
```

Comment:
This is nice but you're not actually using it as you have handled your errors in your app.js?

Personally I would delete the error handling in the app and import this in but you can do it either way just be consistent.